pkg/users/local.js

```diff
@@ -136,7 +136,7 @@ function passwd_change(user, new_pass) {
             dfd.resolve();
         })
         .fail(function(ex, response) {
-            if (ex.constructor.name == "ProcessError") {
+            if (ex.constructor.name == "ProcessError" || ex.exit_status) {
```

Comment:
Why not just ```"exit_status" in ex``` and drop use of ProcessError altogether?

Reply:
Was just trying to keep the old behavior around in case it actual worked for anyone. But it most likely didn't for any meaningful deployments so i removed.